libraries/botframework-streaming/src/interfaces/INodeSocket.ts

```diff
@@ -14,5 +14,38 @@
 export interface INodeSocket {
     writable: boolean;
     write(str: string, cb?: Function): boolean;
+    readable: boolean;
```

Comment:
What's up with this piece of the diff?

Reply:
This was due to some issues with node types being fixed (https://github.com/microsoft/botbuilder-js/pull/2515), but I think some extra changes might have been picked so I'll test backing them out.